docs/contributing/dependencies/index.md

```diff
@@ -9,6 +9,43 @@ Bitwarden uses [Renovate](https://www.mend.io/renovate/) for automating dependen
 will automatically create pull requests for dependencies on a weekly cadence. Security updates will
 generate pull requests immediately.
 
+## Renovate configuration
+
+Renovate is configured by a `.github/renovate.json` (or `.github/renovate.json5`) file in each
+repository. We follow an internal template for consistency. The template is available in the
+[template repository](https://github.com/bitwarden/template/blob/main/.github/renovate.json).
+
+It is recommended that all repositories extend the
+[default](https://github.com/bitwarden/renovate-config/blob/main/default.json) configuration from
+our [shared configuration repository](https://github.com/bitwarden/renovate-config), which is
+included when a new repository is created from the template.
+
+The default configuration:
+
+- Combines minor and patch changes into one rollup pull request, per package manager.
+- Uses a dependency dashboard so we can see what pull requests are not yet created but still manage
+  the workload.
+- Manages updates with semantic versioning and lock file updates. Rebases are disabled.
+- Allows an unlimited number of pull requests to be created.
+- Includes major updates (the latest) as individual pull requests. Monorepo updates are also
+  grouped.
+- Schedules runs to happen on the weekend when more Actions workers are likely available for the
+  organization, but also on a two-week basis to better align with the release schedule.
+- Certain build pipeline dependencies are pinned to specific versions.
+
+See [Management Strategies](#management-strategies) below for more detail on why we have chosen
+these configuration options.
+
+All package managers are recommended to be left enabled should a repository expand over time to
```

Comment:
ℹ️ I don't do this personally, and remove unused managers -- some of them often don't make sense together e.g. Cargo and NPM. Maybe we do something to the template repo to change this? 🤷

Reply:
Can just leave it for now. I always expect some customization really.